core/src/main/scala/org/broadinstitute/dsp/workbench/hamm/db/JobTable.scala

```diff
@@ -106,16 +106,16 @@ object JobBinders {
     def apply(rs: ResultSet, index: Int): WorkflowId = WorkflowId(rs.getString(index))
   }
 
-  implicit val callFqnTypeBinder: TypeBinder[CallFqn] = new TypeBinder[CallFqn] {
-    def apply(rs: ResultSet, label: String): CallFqn = CallFqn(rs.getString(label))
-    def apply(rs: ResultSet, index: Int): CallFqn = CallFqn(rs.getString(index))
+  implicit val callNameTypeBinder: TypeBinder[CallName] = new TypeBinder[CallName] {
```

Comment:
I don't think it's a good idea to name this `CallName`. Like Jeff said, it's from cromwell's metadata, I think it's less confusing to call it the same way